jetty-server/src/test/java/org/eclipse/jetty/server/ConnectorTimeoutTest.java

```diff
@@ -610,11 +611,17 @@ public void testMaxIdleNoRequest() throws Exception
         }
         catch(SSLException e)
         {
-            e.printStackTrace();
+            if(Log.getLog().isDebugEnabled())
+                Log.getLog().debug(e);
+            else
+                Log.getLog().info(e.getMessage());
```

Comment:
Replace here and in the hunks below with a proper log instance allocated in the test class:

```java
public abstract class ConnectorTimeoutTest extends HttpServerTestFixture
{
    protected static final Logger LOG = Log.getLogger(ConnectorTimeoutTest.class);
    ...
}
```

and then use the `LOG` instance instead of `Log.getLog()`.